src/models/UsersModel.js

```diff
@@ -0,0 +1,51 @@
+class UsersModel {
+    constructor({
+      name,
+      email,
+      profilePicture = 'http://localhost:8000/images/userProfileImage.jpg', // Default image URL
+      loginType = null,
```

Comment:
What is `loginType`? They are either a user or an admin/teacher. I thought we agreed to use an `isAdmin` field to determine which they are?

Reply:
It's to check how users log in using Google, email, or Facebook.